src/main/java/de/unistuttgart/towercrushbackend/controller/websockets/WebsocketController.java

```diff
@@ -12,73 +12,34 @@
 import org.springframework.messaging.simp.SimpMessagingTemplate;
 import org.springframework.stereotype.Controller;
 
-import java.time.LocalDateTime;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
 
 @Controller
 @Slf4j
 public class WebsocketController {
 
     @Autowired
     SimpMessagingTemplate simpMessagingTemplate;
-    String destination = "/topic/messages";
-    String gameDestination = "/topic/games/";
-
-    ExecutorService executorService =
-        Executors.newFixedThreadPool(3);
-    Map<String, Future<?>> submittedLobbies = new HashMap<>();
-    @MessageMapping("/start/lobbyName/{lobbyName}")
-    public void startGame(@DestinationVariable String lobbyName) {
-        log.info("Started task");
-        MessageWrapper startMessage = new MessageWrapper("Started game: " + lobbyName, Purpose.CHAT_MESSAGE);
-        simpMessagingTemplate.convertAndSend(gameDestination + lobbyName,
-            startMessage);
-        submittedLobbies.put(lobbyName, executorService.submit(() -> {
-            while (true) {
-                MessageWrapper idleMessage = new MessageWrapper(LocalDateTime.now()
-                    + ": doing game: " + lobbyName, Purpose.CHAT_MESSAGE);
-                simpMessagingTemplate.convertAndSend(gameDestination + lobbyName,
-                    idleMessage);
-                Thread.sleep(10000);
-            }
-        }));
+    final String destination = "/topic/messages";
+    final String lobbyDestination = "/topic/lobbies/";
+
+    @MessageMapping("/lobby/{lobby}/team/{team}/player/{player}")
+    public void joinTeam(@DestinationVariable final String lobby, @DestinationVariable final String team, @DestinationVariable final String player) throws JsonProcessingException {
+        log.info("player: " + player + ", joined team: " + team + ", in lobby: " + lobby);
```

Comment:
```suggestion
        log.info("player '{0}' joined team '{1}' in lobby '{2}'", player, team, lobby);
```

Reply:
hm idk if i like this more, becouse its not so nice to read? if its common style you can commit suggestion and resolve conversation tho.

Reply:
1. It's more performant
2. That is the common approach everyone else will suggest you as well
3. > its not so nice to read

Erm… Are you absolutely certain about that? Most people will disagree with you.

Reply:
yes i am, but  ill commit it anyway